.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,6 +7,11 @@ assignees: ''
 
 ---
 
+**ATTENTION**
+
+Please start filing all issues in our new repo: [https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/new/choose.](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/issues/new/choose)
+Issued filed in this repo should only be bug reports for Spring Cloud GCP 1.x.
```

Comment:
Maybe add in parentheses `(corresponding to Spring Cloud Hoxton or below)`.

Reply:
Done.